assets/jsx/bagger.jsx

```diff
@@ -17,8 +17,19 @@ class Bagger extends React.Component {
     }
 
     componentDidMount() {
-        const { dispatch, uploader: {accessKeyId, secretAccessKey, bucket, region}} = this.props
-        dispatch(BagActions.testConfiguration(accessKeyId, secretAccessKey, bucket, region))
+        const { dispatch } = this.props
+        const b = document.getElementById('bagger');
+        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
```

Comment:
Agreed. Much of the functionality is already guarded behind all of those having values elsewhere.

I changed the guard on this line to any-have-values. As it seems there are use cases where we'll want to pass along some but not all values.